### DISCUSSION TITLE
Proxying Flood with Lighttpd

### DISCUSSION THREAD

**Discussion by eightmillion** [jesec/flood#819] [General]
I was struggling to get flood reverse proxied using lighttpd. Flood would never finish loading in the browser until I killed flood on the server. It turns out lighttpd tries to buffer event streams fully before serving them to the client. Adding `server.stream-response-body = 1` to lighttpd.conf completely fixes this issue.

**Comment by trim21** [comment]
ok, add it to wiki https://github.com/jesec/flood/wiki/Using-Flood-behind-a-web-server-(lighttpd)

**Reply by eightmillion** [reply]
Great, thank you!
